dangerfile.js

```diff
@@ -62,8 +62,22 @@ const thanksGifs = [
   'https://media.giphy.com/media/1lk1IcVgqPLkA/giphy.gif', // Cap salute
 ];
 
-// Be nice to our neighbours.
-if (isPrExternal) {
+if (isPrFromDependabot) {
+  markdown(`
+  # Notes for reviewers about Dependabot PRs
+
+  If CI passes, it's usually ok to merge. We should be confident in CI's ability to catch issues.
```

Comment:
I would argue this as sometimes Dependabot PRs just update the dependency in the `package-lock.json` and not update it in the `package.json` if a direct dependency in the project so maybe we need to word this for us to check if the dependency its upgrading is a direct dependency and if so update the package.json

Reply:
If I understand correctly (it's hard to read when you type one huge run-on sentence 😉), you're saying that the Dependabot PRs that only upgrade the lockfile need additional manual work by us before they can be merged?

If so, it sounds like Dependabot is failing to make usable PRs, which is a separate problem. We can't have it making PRs that aren't mergeable, I'd sooner just switch it off entirely.

Reply:
Yes since we changed the settings it will now sometimes just do lock file upgrades for dependencies that are direct in the project.

So we either further down the line need to update manually or in the same PR or when a major gets released.

Reply:
⬆️ hopefully those are better shorter sentences 😉

Reply:
Yeah that feels kind of wrong to me. I've merged loads of PRs that Dependabot made that only changed the lockfile, because I assumed they were ok to merge. 

Dependabot's PRs needs to be treated exactly like ones we'd get from a human. If somebody made a PR that only did part of the job, we would ask them to stop doing that.

Reply:
Yeah that is something and why we need to treat these with care and see if there are other tools available to do this maybe?

Reply:
On my repos, it bumps the `package.json` for direct dependencies 🤔 
https://github.com/georgegillams/browser-scripts/pull/49/files

Reply:
Looking at that repo @georgegillams it looks like you are still running the preview version over the official GH version (no .github/dependabot.yml file) and the default settings on that will do the `package.json` bump but we recently changed it for some reason will need to dig out why which has now caused it.

Or internally they have made a change to Dependabot

Reply:
👍

Reply:
Ok cool, well let's talk about it on Monday and get to the bottom of it. 

I feel like everything that I wrote in the string above is how it **should** be, and if that's not the case we should do something about it so that we **can** be confident in the PRs that Dependabot makes

Reply:
I'm gonna revert this change so as not to block this PR, and we can discuss it separately on Slack